src/dart-2.0.md

```diff
@@ -0,0 +1,138 @@
+---
+layout: default
+permalink: /dart-2.0
+title: Dart 2.0 Updates
+description: How Dart 2.0 is different than 1.x, and how you can prepare.
+---
+
+This page has information about changes that are coming in Dart 2.0.
+
+<aside class="alert alert-info" markdown="1">
+**Flutter:** You don't need to do anything to prepare Flutter code for Dart 2.0.
+If changes become necessary, watch this page for updates.
+</aside>
```

Comment:
I'd move this below the list (just above "What's different") and rephrase it slightly. Maybe:

**Flutter developer?**
You ... necessary, we'll add Flutter details to this page.

(I can't think of a good way to phrase it without "we" and future tense. Basically, I was trying to avoid the "if something happens that you could learn about by watching this page, watch this page" conundrum.)